### PR TITLE
Add support for keebsforall freebird75

### PR DESCRIPTION
**Pull request by e11i0t23**
## Description
Add support for keebsforall freebird75

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/keebsforall/freebird75/config.h [hunk 1]
```diff
@@ -0,0 +1,20 @@
+// Copyright 2022 Elliot Powell (@Elliot Powell)
```
Comment (zvecr) [inline keyboards/keebsforall/freebird75/config.h:1]: This file can be removed

File: keyboards/keebsforall/freebird75/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,14 @@
+#include QMK_KEYBOARD_H
```
Comment (zvecr) [inline keyboards/keebsforall/freebird75/keymaps/default/keymap.c:1]:
```suggestion
// Copyright 2022 Elliot Powell (@Elliot Powell)
// SPDX-License-Identifier: GPL-2.0-or-later

#include QMK_KEYBOARD_H
```

File: keyboards/keebsforall/freebird75/keymaps/via/keymap.c [hunk 3]
```diff
@@ -0,0 +1,41 @@
+#include QMK_KEYBOARD_H
```
Comment (zvecr) [inline keyboards/keebsforall/freebird75/keymaps/via/keymap.c:1]:
```suggestion
// Copyright 2022 Elliot Powell (@Elliot Powell)
// SPDX-License-Identifier: GPL-2.0-or-later

#include QMK_KEYBOARD_H
```

File: keyboards/keebsforall/freebird75/readme.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+# Freebird 75
+
+![fb75](https://cdn.shopify.com/s/files/1/0466/1408/5790/products/Scene_03_1200x.jpg?v=1664496240)
+
+*A short description of the keyboard/project*
```
Comment (zvecr) [inline keyboards/keebsforall/freebird75/readme.md:5]: Fill in or remove

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/keebsforall/freebird75/readme.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
+# Freebird 75
+
+![fb75](https://cdn.shopify.com/s/files/1/0466/1408/5790/products/Scene_03_1200x.jpg?v=1664496240)
```
Comment (drashna) [inline keyboards/keebsforall/freebird75/readme.md:3]: Please rehost image on Imgur (or another dedicated image hosting site).  Shopify links are non-permanent.

File: keyboards/keebsforall/freebird75/readme.md (hunk 4 above)
Comment (e11i0t23) [reply keyboards/keebsforall/freebird75/readme.md]: Filled in

File: keyboards/keebsforall/freebird75/keymaps/default/keymap.c (hunk 2 above)
Comment (e11i0t23) [reply keyboards/keebsforall/freebird75/keymaps/default/keymap.c:4]: Updated

File: keyboards/keebsforall/freebird75/readme.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
+# Freebird 75
+
+![fb75](https://i.imgur.com/fGRQZ2g.jpg)
```
Comment (drashna) [inline keyboards/keebsforall/freebird75/readme.md:3]:
```suggestion
![fb75](https://i.imgur.com/fGRQZ2gh.jpg)
```

**Comment by e11i0t23** [comment]
Any update here getting this merged? @zvecr

**Review by keyboard-magpie — APPROVED** [review]

File: keyboards/keebsforall/freebird75/info.json [hunk 7]
```diff
@@ -0,0 +1,463 @@
...
+    },
+    "processor": "atmega32u4",
+    "url": "",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x7575",
+        "vid": "0x6B66"
+    },
+    "layouts": {
+        "LAYOUT_75": {
```
Comment (drashna) [inline keyboards/keebsforall/freebird75/info.json:27]: This should match one of the community layouts, or should be left as just `LAYOUT`.
```suggestion
        "LAYOUT": {
```

Either way, that necessitates a change, and needs to be applied to the keymap.c files, as well.

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

**Pull request merged**